fcrepo-search-api/src/main/java/org/fcrepo/search/api/SearchParameters.java

```diff
@@ -17,19 +17,59 @@
  */
 package org.fcrepo.search.api;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * A pojo encapsulating the parameters of a search
  *
  * @author dbernstein
  */
 public class SearchParameters {
 
-    private Query query;
+    private List<Condition> conditions;
+
+    private int offset = 0;
+
+    private int maxResults = 100;
+
+    /**
+     * Constructoor
+     *
+     * @param conditions The conditions
+     * @param maxResults The max results
+     * @param offset     The offset
+     */
+    public SearchParameters(final ArrayList<Condition> conditions, final int maxResults, final int offset) {
```

Comment:
This should be `List` rather than `ArrayList`.

Reply:
Yes - will fix.